spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -536,8 +536,8 @@ If a compensating activity is brief then the synchronous model may be appropriat
     @Path("/compensate")
     @PUT
     public Response compensatePreviousAction(
-            @HeaderParam(LRA_HTTP_CONTEXT_HEADER) String lraId) {
-        getActivity(lraId).setCompensated();
+            @HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
+        getActivity(lraId).setCompensated();  // business logic for handling the LRA, not provided by spec or implementation.
```

Comment:
I would prefer that we replace this with a comment (like you did in the other examples) because someone on the forums thought it was part of the spec API. Similarly for the other occurrence of getActivity the next example.

Reply:
See https://github.com/eclipse/microprofile-lra/issues/168 for updating this.

Reply:
I don't see why you need to raise a separate issue for it. Just change here.

Reply:
I don't like mixing multiple types of changes under one PR.

Anyway, I have fixed the typo and made sure that every example in the Spec document indicates when code statements are not from API or Implementation with a comment.

Anything else needed for this @mmusgrov ?